gfx/Mushroom-Dream/tile_info.json

```
@@ -20,6 +20,12 @@
     "large.png": { "sprite_width": 64, "sprite_height": 64, "sprite_offset_x": -32, "sprite_offset_y": -32 }
   },
   {
-    "enormous.png": { "sprite_width": 128, "sprite_height": 128, "sprite_offset_x": -64, "sprite_offset_y": -96 }
+    "wide tall.png": { "sprite_width": 48, "sprite_height": 64, "sprite_offset_x": -16, "sprite_offset_y": -32 }
```

```suggestion
    "wide-tall.png": { "sprite_width": 48, "sprite_height": 64, "sprite_offset_x": -16, "sprite_offset_y": -32 }
```
I think the extra `_` in the middle will mess up with the script, you should probazbly rename the folder to `pngs_wide-tall_48x64`